web/cypress/integration/plugin/create-edit-delete-plugin.spec.js

```diff
@@ -70,6 +75,6 @@ context('Create and Delete Plugin List', () => {
     });
 
     // check if plugin list is empty
-    cy.get(domSelectors.empty);
+    cy.get(domSelector.empty);
```

Comment:
get then what? How to check it's empty?

Reply:
![image](https://user-images.githubusercontent.com/72343596/105689924-5262a000-5f36-11eb-95a1-ab81588dfe84.png)
It can be detected when there is no data

Reply:
how about add assert `.should('be.visible')` check empty?

Reply:
Don't need `.get().should(xxx)`?

Reply:
I agree. Thank you for your suggestions！